src/transaction_builder.js

```diff
@@ -7,7 +7,7 @@ import { Keypair } from './keypair';
 import { Transaction } from './transaction';
 import { Memo } from './memo';
 
-const BASE_FEE = 100; // Stroops
+export const BASE_FEE = 100; // Stroops
```

Comment:
Any merit in relaxing the constant nature of this until we implement [#233](https://github.com/stellar/js-stellar-sdk/issues/233)? I saw a comment earlier about users having to hack the hard-coded value in the SDK.

Reply:
Users shouldn't change this value; if they don't want to use it, they should use their own constant or variable.